### PR TITLE
[InstCombine] Precommit test for #79417. NFC.

### PR DESCRIPTION
**Pull request by bzEq**

### CODE REVIEW

File: llvm/test/CodeGen/PowerPC/absdiff.ll [hunk 1]
```diff
@@ -0,0 +1,19 @@
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py UTC_ARGS: --version 4
+; RUN: opt -mtriple=powerpc64-linux-gnu -passes=instcombine < %s -o - | llc -mtriple=powerpc64-linux-gnu -o - | FileCheck %s
+
+define  i64 @absdiff(i64 %0, i64 %1) {
```
Comment (ecnelises) [inline llvm/test/CodeGen/PowerPC/absdiff.ll:4]: Will the combination differ by triple? If this is `opt` change, I don't think we should mix `opt` runlines in codegen test to 'prove' combine result is better.

**Review by RKSimon — CHANGES_REQUESTED** [review]
Please don't put instcombine tests in codegen

**Comment by bzEq** [comment]
Put codegen comparison in #79417 .

**Comment by ecnelises** [comment]
If this IR sequence doesn't change after instcombine currently, you can just add it in the patch. Also, maybe you can put it in files like `abs-1.ll` or `abs_abs.ll`.

**Comment by bzEq** [comment]
Put tests directly in #79417.

**Pull request closed without merging**